docs/user/05-60-kyma-bindings.md

```diff
@@ -16,32 +31,87 @@ X-Broker-API-Version: 2.14
+
+
+<!-- Binding Creation -->
+In the binding creation phase, KEB creates a Service Binding object and generates a kubeconfig file with a JWT token. The kubeconfig file is valid for a specified time period, which is defaulted of set in the request body. The first step in this part is to check again if an expired binding exists in the database. This check is done in an implicit DB insert statement. The query will fail for expired but existing bindings because of primary key being defined on the instance and binding IDs and not expiration date. This will be the case until the expired binding is removed from the database by the cleanup job. 
+> **Note:** Expired bindings do not count towards the bindings limit, however they will prevent from creating new bindings until they exist in the database. Only after they are removed by the cleanup job or manually, the binding can be recreated again.
+
+After the insert has been done, KEB creates a ServiceAccount, ClusterRole (admin privileges), and ClusterRoleBinding, all named `kyma-binding-{{binding_id}}`. The ClusterRole can be used to modify permissions granted to the kubeconfig.
+
+The created resources are then used to generate a [TokenRequest](https://kubernetes.io/docs/reference/kubernetes-api/authentication-resources/token-request-v1/). The token is then wrapped in a kubeconfig template and returned to the user. The encrypted credentials are then stored as an attribute in the previously created database binding.
+
+> **NOTE**: Creation of multiple and unused TokenRequests is not recommended
+
+
+### Fetching Kyma Binding Process
+
+![Get Binding Flow](../assets/bindings-get-flow.drawio.svg)
+
+The above diagram shows a flow of fetching a Kyma Binding in KEB. The process starts with a GET request sent to the KEB API. Bindings are located by instance and binding IDs. The first instructions in the process is to check if Kyma instance exists, and if it exists, then it must not be deprovisioned or suspended. The endpoint will not return bindings for such instances. Existing Bindings are loaded by instance ID and binding ID. If any bindings exists they are filter by expiration date. KEB returns only non expired bindings.
```

Comment:
```suggestion
The above diagram shows a flow of fetching a Kyma Binding in KEB. The process starts with a GET request sent to the KEB API. Bindings are identified by instance and binding IDs. The first instructions in the process is to check if Kyma instance exists, and if it exists, then it must not be deprovisioned or suspended. The endpoint will not return bindings for such instances. Existing Bindings are loaded by instance ID and binding ID. If any bindings exists they are filter by expiration date. KEB returns only non expired bindings.
```

Reply:
Existing Bindings are retrieved by instance ID and binding ID